### PR TITLE
Create pure-writer.rb

### PR DESCRIPTION
**Pull request by peng1999**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @peng1999.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by peng1999** [comment]
> Thank you for your contribution to `homebrew-cask` @peng1999.
> 
> Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

This app may not have any settings to be zapped.
I run `brew cratezap pure-writer` and it says `No pure-writer settings found.`

**Comment by peng1999** [comment]
> > Thank you for your contribution to `homebrew-cask` @peng1999.
> > Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.
> 
> This app may not have any settings to be zapped. I run `brew cratezap pure-writer` and it says `No pure-writer settings found.`

I'm sorry, I found the preference file `~/Library/Preferences/com.drakeet.purewriter.plist`. I'll add the zap at once.

File: Casks/purewriter.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cask "pure-writer" do
+cask "purewriter" do
```
Comment (peng1999) [inline Casks/purewriter.rb:1]: Although the pkg file is named `PureWriter-1.4.0.pkg`, the app it installs is in `/Applications/Pure Writer.app`.
According to the token reference:
> The Simplified Name of a pkg may be more tricky to determine than that of an App. If a pkg installs an App, then use that App name with the rules above. If not, just create the best name you can, based on the vendor’s web page.

So why didn't we add a slash between words, following the app name it installs?

**Comment by miccal** [comment]
Thank you @peng1999.

**Pull request merged**